script/cibuild

```diff
@@ -143,11 +198,7 @@ build_scheme ()
         action=build
     fi
 
-    run_xctool $sdkflag -scheme "$scheme" $action
+    run_build_tool $sdkflag -scheme "$scheme" $action
 }
 
-export -f build_scheme
-export -f run_xctool
-export -f parse_build
-
-main
+main
```

Comment:
I updated those submodules to the latest commits in the `swift-2.0` branches a few times. It looks like Nimble has since merged that into `master`, but Quick hasn't yet.